### DISCUSSION TITLE
electricity consumption

### DISCUSSION THREAD

**Discussion by Lorizzo-Mirco** [petretiandrea/home-assistant-tapo-p100#367] [Q&A]
from the home assistant you can read the consumption of a tapo p110 socket

**Comment by Angrypeanut** [comment]
I'm not sure if this is what Supermicro is meaning
But about a month or so ago my Tapo P110 stopped showing "Current Energy" usage but all the other datasets are still working perfectly.
I go to the Tapo app and the all data is available in there including "Current Power"

Wondering if they changed naming which broke this or something else as it was working and then stopped.

Thank you again for your time and effort which makes my HA install so awesome

Not sure why they are different errors but exact same device


![Screenshot_20230302_074207](https://user-images.githubusercontent.com/94326553/222391514-e22c7ed9-1e7f-4ea4-b347-1a775b4063de.png)
![Screenshot_20230302_074220](https://user-images.githubusercontent.com/94326553/222391524-60975795-708a-43e8-a58a-56ab3ac14c9e.png)
